lib-sql/functions/importance.sql

```diff
@@ -125,7 +125,8 @@ BEGIN
     END LOOP;
   END IF;
 
-  RETURN null;
+  result.importance := 0.75 - (rank_search / 40);
```

Comment:
There have been issues with artihmetics when the importance was exactly 0. Could you change the formula to using 0.750001, so we never hit the 0?